### ISSUE TITLE
Livereload

### ISSUE THREAD

**Issue by tatecarson** [sebpiq/rhizome#142]
How would I use rhizome with a tool that does live reloading like forever or nodemon?

**Comment by sebpiq** [comment]
Hi @tatecarson ! What do you mean by livereload? Do you mean automatic restart if the server crash?

**Comment by lisongx** [comment]
Did you mean when you developing, the server automatic reload the code when your code change?

**Comment by tatecarson** [comment]
Yes, for development. It's a small thing but very helpful in development. I can't figure out how to get it to work because rhizome is called differently than other node applications. For instance, https://nodemon.io/ will watch your source for changes and restart the app. I'm just not sure how to use it with rhizome.

**Comment by sebpiq** [comment]
so the thing is rhizome is just a node.js file. You need to find where it gets installed on your system by running `which rhizome` and then you can run it through `nodemon` instead of calling it directly. For example : 

`nodemon /full/path/to/rhizome.js config`

Tell me if it helps!
